File: xk/XueKao.Tool/XueKao.Tool/Model/Enum/CouchBaseSectionType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace XueKao.Tool.Model.Enum
{
    /// <summary>
    /// CouchBase的Bucket类别，一般以Bucket名称作为枚举值，前缀可以去掉，例如Motk-Common可用使用Common表示
    /// </summary>
    public enum CouchBaseSectionType
    {
        Main = 1,
        Examresult = 2,
    }
}

File: xk/xk_common/src/main/java/com/ry/xk/springbootframe/couchbase/CouchBaseSectionType.java
package com.ry.xk.springbootframe.couchbase;

public enum CouchBaseSectionType {
    MAIN(1), EXAMRESULT(2);
    private int index;

    CouchBaseSectionType(int index)
    {
        this.index = index;
    }

    public int getIndex()
    {
        return this.index;
    }
}
